homeassistant/components/smartthings/strings.json

```
@@ -242,6 +242,9 @@
           "finished": "[%key:component::smartthings::entity::sensor::oven_job_state::state::finished%]"
         }
       },
+      "cooler_temperature": {
+        "name": "Cooler temperature"
```

isn't it rather Refrigerator? I think a cooler usually consumes no power, but is rather just a box
```suggestion
        "name": "Refrigerator temperature"
```

---

I think I have this merged somewhere else already, do you mind merging this and checking with the smartthings users in the meantime?

---

ok